db/migrate/20170728231236_modify_mhv_ac_terms_content.rb

```diff
@@ -0,0 +1,27 @@
+class ModifyMhvAcTermsContent < ActiveRecord::Migration
+  def up
+    yes_content = 'Yes, I agree with the terms and conditions for medical information'
+
+    terms_and_conditions = TermsAndConditions.find_by(
+      name: 'mhvac',
+      version: '1.0'
+    )
```

Comment:
minor nitpick, but these two could be DRY'd up by moving the lookup into a method, but since this is just a migration, it's really not a big deal at all.